pkg/cli/cli.go

```diff
@@ -121,6 +122,7 @@ Load noobaa completion to bash:
 			obc.Cmd(),
 			diagnose.Cmd(),
 			system.CmdUI(),
+			dbdump.Cmd(),
```

Comment:
Why do we need a whole command for it if it is in the diag?

Reply:
The issue related to this PR requested an additional command. One of the comments mentioned it can be added to the diagnose command using a flag - I implemented accordingly.